packages/saas-ui-auth/src/components/forgot-password-form.tsx

```diff
@@ -38,7 +38,7 @@ export const ForgotPasswordForm: React.FC<ForgotPasswordFormProps> = ({
   onError = () => null,
   onValidationError,
   submitLabel,
-  label,
```

Comment:
Good idea to rename these! But let's keep them in here, as I don't want to introduce breaking changes in a minor update.

Add a comment to deprecate them and fallback to label if emailLabel isn't set.
```
/**
 * @deprecated
 * /
 label?: string
```

Reply:
Applies to all labels that have been renamed 🙏

Reply:
It was implied 👍 please check which fallback order you prefer `new ?? old` vs `old ?? new`